spotseeker_server/test/auth/oauth.py

```diff
@@ -320,3 +322,20 @@ def test_create_trusted_client(self):
         client = TrustedOAuthClient.objects.get(consumer=consumer)
 
         self.assertIsInstance(client, TrustedOAuthClient)
+
+    @override_settings()
```

Comment:
This is apparently the documentation way of temporarily deleting settings attributes